src/Psy/CodeCleaner/ValidFunctionNamePass.php

```diff
@@ -58,7 +59,7 @@ public function leaveNode(Node $node)
         if ($node instanceof FuncCall) {
             // if function name is an expression or a variable, give it a pass for now.
             $name = $node->name;
-            if (!$name instanceof Expression && !$name instanceof Variable) {
+            if (!$name instanceof Expr && !$name instanceof Variable) {
```

Comment:
I think it's a bug, because PHP parser doesn't have `Expression` type. It has only `Expr`.